lib/events-reader.js

```diff
@@ -116,6 +116,38 @@ module.exports = function (harvesterApp) {
 
                         var dfd = RSVP.defer();
 
+                        function checkFilterExecuteHandler(id, dfd, opFn, changeHandlerOp, doc) {
+                            var filter = 'o.$set.' + opFn.filter;
```

Comment:
I *think* that this will not work for inserts, but will work for updates. This is because inserts have a 0=>inserted object (e.g.  {equipment:"1212321e121c12def", externalId:"21",model:"1212321e121c12dee"}, when updates have o.$set=>update (e.g. {equipment:"1212321e121c12def"}